packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx

```diff
@@ -329,6 +332,9 @@ const SidebarNavigation: React.FC = () => {
             <SidebarAnchor level={2} to={`${SidebarCategoryRoutes.LIBRARIES}/codemods`}>
               Codemods
             </SidebarAnchor>
+            <SidebarAnchor level={2} to={`${SidebarCategoryRoutes.LIBRARIES}/data-visualization`}>
+              Data visualization
```

Comment:
thought (non-blocking): I wonder if having two pages with the same name would be confusing. That said, I can't think of alternative naming so this is probably fine unless someone else has a suggestion. Leaving this comment for future readers to know that yes, we considered it.

Reply:
I thought about this for a while lol. I could call it "data visualization library" but it felt long.

Reply:
#GoodEnough? 🤷🏻‍♂️